frontend/workflows/envoy/src/remote-triage/index.tsx

```diff
@@ -59,10 +59,20 @@ const TriageIdentifier: React.FC<TriageChild> = ({ host = "" }) => {
   );
 };
 
+const downloadFile = (filename: string, blob: Blob) => {
+  const element = document.createElement("a");
+  element.href = URL.createObjectURL(blob);
+  element.setAttribute("download", filename);
+  element.style.display = "none";
+  document.body.appendChild(element);
+  element.click();
+  document.body.removeChild(element);
+};
+
```

Comment:
I'd suggest https://www.npmjs.com/package/file-saver if you didn't try it/run into issues. i think it might simplify things; the internal Aggrefest Viewer is a good example as it does something similar and uses the package